### DISCUSSION TITLE
Cormorant Roaster

### DISCUSSION THREAD

**Discussion by aljamea** [artisan-roaster-scope/artisan#619] [General]
Hi GitHub Community
I am so excited to use and profiling my roaster with artisan software, Just want to be sure is...
Can I use it for my CR600 (Cormorant Roasters)?
I Try it last time but it didn't work for me, Not sure why!
I found on the official site of Cormorant Roasters its Compatible, as you see in the link below
https://www.cormorantroasters.co.uk/specifications/
anyone have experience with it, please 
Thanks in advance

**Comment by CarefreeBuzzBuzz** [comment]
Yes it works.  Read the [Quick Start Guide](https://artisan-scope.org/docs/quick-start-guide/) to understand  how to connect your devices and begin to use Artisan.  If you still have issues,  then be very specific about what is happening.   Your inquiry is to vague to help.   You must put your hardware correctly, install drivers if you are using Phidgets, and so some basic reading and learning first.  Thanks for being here.
